frontend/src/components/AriaLiveRegion.js

```diff
@@ -5,7 +5,7 @@ function AriaLiveRegion(props) {
   const { messages } = props;
 
   return (
-    <div className="sr-only" role="status">
+    <div className="usa-sr-only" role="status">
```

Comment:
`usa-sr-only` comes from the [USWDS design system](https://github.com/uswds/uswds/search?q=usa-sr-only), though it isn't well documented.  Not sure where the plain `sr-only` comes from (though it had worked).